### PR TITLE
[ADD] sale_quotation_auto_confirm

### PR DESCRIPTION
**Pull request by TimLai125**

### CODE REVIEW

File: sale_quotation_auto_confirm/models/sale_order.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        threshold_date = (datetime.today() + timedelta(days=7)).strftime(
+            '%Y-%m-%d %H:%M:%S')
+        pending_quotations = self.search([
+            ('state', 'in', ('sent', 'draft')),
+            ('team_id', '=', self.env.ref(
+                'sales_team.team_sales_department').id),
+            '|',
+            ('requested_date', '<=', threshold_date),
+            ('requested_date', '=', False),
+            ('commitment_date', '<=', threshold_date),
```
Comment (TimLai125) [inline sale_quotation_auto_confirm/models/sale_order.py]: @yostashiro Is the domain here fine?
i.e. Sales Orders that are having 
( status in 'sent' or 'draft' ) and 
( sales team is 'Direct Sales' ) and 
(
	( Requested Date is after 7 days or more than the current date ) or 
	(( Requested Date is empty ) and ( Commitment Date is after 7 days or more than the current date))
)

File: sale_quotation_auto_confirm/models/crm_team.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+from odoo import models, fields
+
+
+class CrmTeam(models.Model):
+    _inherit = 'crm.team'
+
+    auto_confirm_sale_order = fields.Boolean(
+        string='Auto Confirm Sales Order by Scheduled Action',
+        store=True,
```
Comment (yostashiro) [inline sale_quotation_auto_confirm/models/crm_team.py]: No need to have `store=True,` here.

File: sale_quotation_auto_confirm/models/sale_order.py [hunk 3]
```diff
@@ -12,16 +12,20 @@ class SaleOrder(models.Model):
...
+                days=threshold_days)).strftime('%Y-%m-%d %H:%M:%S')
+            pending_quotations = self.search([
+                ('state', 'in', ('sent', 'draft')),
+                ('team_id', '=', sale_team.id),
+                '|',
+                ('requested_date', '<=', threshold_date),
+                ('requested_date', '=', False),
+                ('commitment_date', '<=', threshold_date),
+            ])
+            print pending_quotations
```
Comment (yostashiro) [inline sale_quotation_auto_confirm/models/sale_order.py]: Remove this line.

**Review by yostashiro — CHANGES_REQUESTED** [review]

File: sale_quotation_auto_confirm/models/sale_order.py (hunk 3 above)
Comment (TimLai125) [reply sale_quotation_auto_confirm/models/sale_order.py]: ok, I will remove it

File: sale_quotation_auto_confirm/models/crm_team.py (hunk 2 above)
Comment (TimLai125) [reply sale_quotation_auto_confirm/models/crm_team.py]: We need to use this field to search for `sales_team`, doesn't mean we need to store it?

**Pull request merged**